rotkehlchen/chain/evm/decoding/aave/v3/constants.py

```diff
@@ -10,3 +11,4 @@
 BORROW: Final = b'\xb3\xd0\x84\x82\x0f\xb1\xa9\xde\xcf\xfb\x17d6\xbd\x02U\x8d\x15\xfa\xc9\xb0\xdd\xfe\xd8\xc4e\xbcsY\xd7\xdc\xe0'  # noqa: E501
 REPAY: Final = b'\xa54\xc8\xdb\xe7\x1f\x87\x1f\x9f50\xe9zt`\x1f\xea\x17\xb4&\xca\xe0.\x1cZ\xeeB\xc9lx@Q'  # noqa: E501
 BURN: Final = b'L\xf2[\xc1\xd9\x91\xc1u)\xc2R\x13\xd3\xcc\x0c\xda)^\xea\xad_\x13\xf3a\x96\x9b\x12\xeaH\x01_\x90'  # noqa: E501
+DEBT_TOKEN_SYMBOL_REGEX: Final = re.compile(r'^(stableDebt|variableDebt)\w+$')
```

Comment:
Have you checked if in all the chains it is the same?

Reply:
yes, it seems. Doing this change and then running `rotkehlchen/tests/unit/test_tasks_manager.py::test_maybe_update_aave_v3_underlying_assets`, only prints the ATokens.
![image](https://github.com/rotki/rotki/assets/55619686/f4be6872-be84-4b47-bad6-3e1f4b6f3180)
```
aEthWETH
aEthwstETH
aEthWBTC
aEthUSDC
aEthDAI
aEthLINK
aEthAAVE
aEthcbETH
aEthUSDT
aEthrETH
aEthLUSD
aEthCRV
aEthMKR
aEthSNX
aEthBAL
aEthUNI
aEthLDO
aEthENS
aEth1INCH
aEthFRAX
aEthGHO
aEthRPL
aEthsDAI
aEthSTG
aEthKNC
aEthFXS
aEthcrvUSD
aEthPYUSD
aOptDAI
aOptLINK
aOptUSDC
aOptWBTC
aOptWETH
aOptUSDT
aOptAAVE
aOptSUSD
aOptOP
aOptwstETH
aOptLUSD
aOptMAI
aOptrETH
aOptUSDCn
aPolDAI
aPolLINK
aPolUSDC
aPolWBTC
aPolWETH
aPolUSDT
aPolAAVE
aPolWMATIC
aPolCRV
aPolSUSHI
aPolGHST
aPolBAL
aPolDPI
aPolEURS
aPolJEUR
aPolAGEUR
aPolMIMATIC
aPolSTMATIC
aPolMATICX
aPolwstETH
aPolUSDCn
aArbDAI
aArbLINK
aArbUSDC
aArbWBTC
aArbWETH
aArbUSDT
aArbAAVE
aArbEURS
aArbwstETH
aArbMAI
aArbrETH
aArbLUSD
aArbUSDCn
aArbFRAX
aArbARB
aBasWETH
aBascbETH
aBasUSDbC
aBaswstETH
aBasUSDC
aGnoWETH
aGnowstETH
aGnoGNO
aGnoUSDC
aGnoWXDAI
aGnoEURe
aGnosDAI
aScrWETH
aScrUSDC
aScrwstETH
```